### PR TITLE
[tests-only][full-ci] Remove the tests already present in middleware from sharingContext

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR removes the tests step already present in middleware and refactors the existing test step to use middleware's and bumps middleware's version to `v1.2.0`

## Related Issue

- Part of https://github.com/owncloud/web/issues/6111

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortraitNotifications</strong> https://drone.owncloud.com/owncloud/web/21576/42/1
The following scenarios passed on retry:
- webUISharingNotifications/shareWithUsers.feature:21
- webUISharingNotifications/shareWithUsers.feature:40

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingIntUsers1</strong> https://drone.owncloud.com/owncloud/web/21576/30/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:53

**Comment by ownclouders** [comment]
Results for <strong>oC10iPhone3</strong> https://drone.owncloud.com/owncloud/web/21576/49/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:48
- webUIUpload/upload.feature:90

**Comment by ownclouders** [comment]
Results for <strong>oC10FileOperations</strong> https://drone.owncloud.com/owncloud/web/21576/14/1
The following scenarios passed on retry:
- webUICreateFilesFolders/createFolderEdgeCases.feature:19

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic1</strong> https://drone.owncloud.com/owncloud/web/21576/12/1
The following scenarios passed on retry:
- webUIAccount/accountInformation.feature:34

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublic2</strong> https://drone.owncloud.com/owncloud/web/21576/38/1
The following scenarios passed on retry:
- webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:60

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp2</strong> https://drone.owncloud.com/owncloud/web/21576/69/1
The following scenarios passed on retry:
- webUIUpload/upload.feature:22

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublic1</strong> https://drone.owncloud.com/owncloud/web/21576/37/1
The following scenarios passed on retry:
- webUISharingPublicBasic/publicLinkCreate.feature:28

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPermission</strong> https://drone.owncloud.com/owncloud/web/21576/35/1
The following scenarios passed on retry:
- webUISharingPermissionsUsers/sharePermissionsUsers.feature:21
- webUISharingPermissionsUsers/sharePermissionsUsers.feature:52

**Comment by ownclouders** [comment]
Results for <strong>oC10Files3</strong> https://drone.owncloud.com/owncloud/web/21576/18/1
The following scenarios passed on retry:
- webUIRenameFiles/renameFiles.feature:39

**Comment by ownclouders** [comment]
Results for <strong>oC10Notification</strong> https://drone.owncloud.com/owncloud/web/21579/39/1
The following scenarios passed on retry:
- webUISharingNotifications/shareWithGroups.feature:24

File: tests/acceptance/features/webUISharingInternalUsersToRootPreviews/shareWithUsers.feature [hunk 1]
```diff
@@ -10,10 +10,17 @@ Feature: Sharing details and preview of files
   @issue-2897
   Scenario: sharing details of items inside a shared folder ("via" info)
     Given user "Carol" has been created with default attributes and without skeleton files in the server
+<<<<<<< HEAD
```
Comment (phil-davis) [inline tests/acceptance/features/webUISharingInternalUsersToRootPreviews/shareWithUsers.feature:13]: This file needs conflicts sorted out

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingPublic2</strong> https://drone.owncloud.com/owncloud/web/21649/65/1
The following scenarios passed on retry:
- webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:78

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/21735/68/1
The following scenarios passed on retry:
- webUIPrivateLinks/accessingPrivateLinks.feature:17

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublic1</strong> https://drone.owncloud.com/owncloud/web/21963/37/1
The following scenarios passed on retry:
- webUISharingPublicExpire/shareByPublicLinkExpiringLinks.feature:11

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingIntGroups</strong> https://drone.owncloud.com/owncloud/web/21973/28/1
The following scenarios passed on retry:
- webUISharingInternalGroups/shareWithGroups.feature:73

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/21973/15/1
The following scenarios passed on retry:
- webUISharingAcceptSharesToRoot/acceptShares.feature:108

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/21980/68/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIPrivateLinks-accessingPrivateLinks_feature-L17.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/21980/40/1
The following scenarios passed on retry:
- webUISharingExternal/federationSharing.feature:106

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by phil-davis — APPROVED** [review]

**Pull request merged**